### PR TITLE
skip cuda test_cholesky_solve_batched_many_batches due to illegal memory access

### PR DESCRIPTION
**Pull request by xwang233**
See https://github.com/pytorch/pytorch/issues/48996

### CODE REVIEW

File: test/test_linalg.py [hunk 1]
```diff
@@ -1895,6 +1895,7 @@ def test_cholesky_solve_batched_non_contiguous(self, device, dtype):
             self.assertEqual(x, x_exp)
 
     @slowTest
+    @onlyCPU  # GPU magma has illegal memory access. See https://github.com/pytorch/pytorch/issues/48996
```
Comment (mruberry) [inline test/test_linalg.py:1898]: Use skipCUDAIf like this:

https://github.com/pytorch/pytorch/blob/d307601365c3b848072b8b8381208aedc1a0aca5/test/test_linalg.py#L246

**Review by mruberry — APPROVED** [review]
Thanks @xwang233!

**Comment by ngimel** [comment]
Should we skip this test for complex128 only (per linked issue), or for all the dtypes?

**Comment by xwang233** [comment]
I've tested that the other three dtypes don't have cuda illegal memory access. I'm ok with either.

**Pull request closed without merging**